script/tool/lib/src/native_test_command.dart

```diff
@@ -622,3 +674,15 @@ class _PlatformResult {
   /// Ignored unless [state] is `failed`.
   final String? error;
 }
+
+/// The state of a .java test file.
+class _JavaTestInfo {
+  const _JavaTestInfo(
+      {required this.usesFlutterTestRunner,
+      required this.hasDartIntegrationTestAnnotation});
+
+  /// Whether the test class uses the FlutterTestRunner runner.
```

Comment:
optional: runner seems a little redundant at the end of this sentence.

Reply:
Fixed.